notion/manifest.json

```diff
@@ -0,0 +1,46 @@
+{
+  "manifest_version": "2.0.0",
+  "app_uuid": "0a709534-658c-4d8f-99a3-566dd7cd809b",
+  "app_id": "notion",
+  "display_on_public_website": true,
+  "tile": {
+    "overview": "README.md#Overview",
+    "configuration": "README.md#Setup",
+    "support": "README.md#Support",
+    "changelog": "CHANGELOG.md",
+    "description": "Monitor your Notion workspace events and customize detections in Datadog",
+    "title": "Notion",
+    "media": [
+      {
+        "media_type": "image",
+        "caption": "Setting up the Datadog integration in Notion",
+        "image_url": "images/Notion_DD.png"
+      },
+      {
+        "media_type": "image",
+        "caption": "Notion event logs in Datadog",
+        "image_url": "images/Notion_DD_2.png"
+      }
+    ],
+    "classifier_tags": [
+      "Category::Event Management",
+      "Category::Collaboration",
+      "Supported OS::Any",
+      "Submitted Data Type::Events"
+    ]
+  },
+  "assets": {
+    "dashboards":
+    {"dashboard_short_name": "assets/dashboards/NotionDashboard.json"}
```

Comment:
The dashboard's `short_name` should be more specific to your integration (i.e `notion` or preferably something more specific like `notion_events`)

Reply:
Apologies if there was confusion, I was referring to changing the key here:
e.g.
`dashboard_short_name` -> `notion_events`

Reply:
changed